### DISCUSSION TITLE
Adding SDR Content Brightness Options

### DISCUSSION THREAD

**Discussion by simonboet** [xanderfrangos/twinkle-tray#819] [Ideas]
I am a long time TwinkleTray user and love the application.

I have just gotten my hands on a new HDR display and am saddened that I cannot adjust brightness on the fly with twinkly tray when HDR is on in Windows 11. 

Instead, I have to navigate to System > Display > HDR and adjust the content brightness slider. 

An option in TwinkleTray to show the slider with a monitors other settings would be an absolute killer feature for users like me. 

Here is a Stackoverflow discussion on adjust SDR brightness programatically which may be useful: https://stackoverflow.com/questions/74594751/controlling-sdr-content-brightness-programmatically-in-windows-11
However, it does seem like the findings may be deprecated.

There is another project named ColorControl that has SDR brightness adjustment capability, but it does not seem to be working for me with the latest build.

**Comment by Wordsmith9091** [comment]
I just wanted to add my "here here" to this request. I imagine this isn't trivial to do, because I suspect someone would have done it by now if it was. But if there IS a way to add this feature to TT, it would be incredibly useful.

It's honestly very silly that Windows doesn't natively support any sort of keyboard shortcut to control SDR brightness in HDR mode (but then again, it's silly that it can't support brightness controls for DDC/CI external monitors with the same shortcuts that already exist for controlling laptop monitors).

**Comment by evaera** [comment]
[HDR Sundial](https://github.com/evaera/hdr-sundial) might be useful
